### ISSUE TITLE
Attribute completion sorting broken

### ISSUE THREAD

**Issue by rictic** [Polymer/vscode-plugin#6]
The VS Code 1.6.0 update seems to have changed the way that completions from multiple systems interoperate. Now we're getting completions from both this plugin and from the built in html attribute suggestion system, which is good, but we're losing out on the custom sorting, which is unfortunate.

**Comment by bahrus** [comment]
I don't recall what it did prior to 1.6, but I was about to log a suggestion to put the Polymer properties first, followed by all the generic attributes.  I assume that's what the custom sorting did / will do?

**Comment by rictic** [comment]
Yep. You can get the old behavior back by disabling vscode's HTML suggestions. Set `"html.suggest.html5": false`

I think long term we can include metadata for native elements and their attributes etc and set a default setting value such that installing `polymer-ide` will default ``html.suggest.html5` to  false.
